uno.platform.uno-calculator.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: uno.platform.uno-calculator
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: start-uno-calculator.sh
+
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+modules:
+  - name: lzo
+    config-opts:
+      - "--enable-shared"
+      - "--disable-static"
+    cleanup:
+      - /include
+      - /share/doc
+      - "*.la"
+      - /lib/*.so
+    sources:
+      - type: archive
+        url: "http://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz"
+        sha256: c0f892943208266f9b6543b3ae308fab6284c5c90e627931446fb49b4221a072
+        x-checker-data:
+          type: anitya
+          project-id: 1868
+          url-template: http://www.oberhumer.com/opensource/lzo/download/lzo-$version.tar.gz
+
+  - name: squashfs-tools
+    buildsystem: simple
+    build-commands:
+      - XZ_SUPPORT=1 LZO_SUPPORT=1 make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=${FLATPAK_DEST}/bin INSTALL_MANPAGES_DIR=${FLATPAK_DEST}/man/man1
+    sources:
+      - type: git
+        url: https://github.com/plougher/squashfs-tools.git
+        tag: 4.6.1
+        commit: d8cb82d9840330f9344ec37b992595b5d7b44184
+        x-checker-data:
+          type: git
+          tag-pattern: ^([\\d.]+)$
+
+  - name: uno-calculator
+    buildsystem: simple
+    build-commands:
+      - unsquashfs uno-calculator.snap
+      - mkdir /app/uno-calculator
+      - chmod +x squashfs-root/Calculator.Skia.Gtk
+      - cp -r squashfs-root/* /app/uno-calculator
+      - install -Dm644 "/app/uno-calculator/meta/gui/icon.png" "/app/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png"
+      - install -Dm644 "/app/uno-calculator/meta/gui/uno-calculator.desktop" "/app/share/applications/${FLATPAK_ID}.desktop"
+      - sed -i 's/^Exec=.*/Exec=start-uno-calculator.sh/' "/app/share/applications/${FLATPAK_ID}.desktop"
+      - sed -i 's/^Icon=.*/Icon=uno.platform.uno-calculator/' "/app/share/applications/${FLATPAK_ID}.desktop"
+      - install -Dm644 uno.platform.uno-calculator.appdata.xml "/app/share/metainfo/${FLATPAK_ID}.appdata.xml"
+      - install -Dm755 start-uno-calculator.sh /app/bin/start-uno-calculator.sh
+    sources:
+      - type: file
+        dest-filename: uno-calculator.snap
+        only-arches: [x86_64]
+        url: https://api.snapcraft.io/api/v1/snaps/download/ZZjSue7jjdQXHsNz5RI9BH3Pf3UnRo0V_58.snap
+        sha256: 2687cbbda5af4aeb5b65a2d8c0c645f1993666df98ba608e827770ed0683d518
+        size: 48603136
+        x-checker-data:
+          type: snapcraft
+          name: uno-calculator
+          channel: stable
```

Comment:
Why is this extra-data? The license says Apache-2.0?

Reply:
Sorry I don't understand your comment. I thought what this bit does is check if updates are available? Do I not need it if the app is open source?

Reply:
the `size` propery confused me, that is only needed for extra-data not for other sources.